solidity_contracts/src/CairoPrecompiles/PragmaCaller.sol

```diff
@@ -7,10 +7,43 @@ using CairoLib for uint256;
 
 contract PragmaCaller {
```

Comment:
Add docstring context for the contract + docstring on all functions with params and return type

Reply:
Added 🫡